contrib/backup/zammad_backup.sh

```diff
@@ -6,6 +6,9 @@
 PATH=/sbin:/bin:/usr/sbin:/usr/bin:
 BACKUP_SCRIPT_PATH="$(dirname $(realpath $0))"
 
+# chdir to run as Cron
+cd $BACKUP_SCRIPT_PATH;
```

Comment:
I'd prefer `cd "${BACKUP_SCRIPT_PATH}"` at this point.
Other than that, it's fine to me.

Verified it from my end.